freeipa/src/main/java/com/sequenceiq/freeipa/service/image/CoreImageProvider.java

```diff
@@ -76,6 +76,6 @@
     private Optional<Image> convert(ImageV4Response response) {
         return Optional.ofNullable(response)
                 .map(r -> new Image(r.getCreated(), r.getDate(), r.getDescription(), r.getOs(), r.getUuid(), r.getImageSetsByProvider(), r.getOsType(),
-                        r.getPackageVersions()));
+                        r.getPackageVersions(), response.isAdvertised()));
```

Comment:
Other properties use r instead of response

Reply:
true :-)